### ISSUE TITLE
Ask for 'Validation' during training and the trend about G_GAN, G_L1, D_fake and D_real 

### ISSUE THREAD

**Issue by Paylet** [junyanz/pytorch-CycleGAN-and-pix2pix#288]
Hello, professor. Recentlly I have trained many times with my own T1-T2 weighted MRI dataset on the pix2pix model. First I set the size of pseudo-color image to _512*512_, and the train dataset obtains _3166_ images with the test dataset obtains _990_. Then I crop each image to four part which size is _256*256_, so that I can see whether the input size influence the information distribution, however the results seem similar. I have trained almost 100 epoches each time, but the value of G_GAN, D_fake and D_real remains unchanged between [0 2] all the time, and G_L1 stays around 10（lambda=100）after 10th epoch, is this normal? 
Finally, I hope you could give me an advice about how to do the _validation_ during the training so that I can avoid overfitting and stop the training. Thank you so much. 
![default](https://user-images.githubusercontent.com/28921771/41221311-5791d332-6d96-11e8-991a-4ee69f1b3218.png)
the figure are results in 75 epoch，512*512

**Comment by Gavin666Github** [comment]
look， I met the same situation as you.
![image](https://user-images.githubusercontent.com/35061171/41266211-8ca909fe-6e28-11e8-89c1-b4d496a47581.png)

**Comment by tangjilin** [comment]
Hello, I have the similar situation with you. Have you solved it? I have 10000 images to train my model.
My loss plot is like this:
![my](https://user-images.githubusercontent.com/35884085/42017042-46af0344-7ae0-11e8-8e7c-9d6b71d32fb9.png)

**Comment by Paylet** [comment]
I have improved the accuracy of the input, and the LI_loss reduced to 4. But I can't find a good way about the validation in the tain.

**Comment by carluri** [comment]
Hi @Paylet, were you able to solve your problem?  Also, were you training with the default parameters? or did you change them (e.g. learning rate, batch_size, etc)?   I'm trying to start something similar and I'm also having a similar issue as described by you.  Any help is appreciated. Thank you.
